x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/action/apikey/GrantApiKeyRequest.java

```diff
@@ -14,6 +14,7 @@
 import org.elasticsearch.common.io.stream.StreamOutput;
 import org.elasticsearch.common.io.stream.Writeable;
 import org.elasticsearch.common.settings.SecureString;
+import org.elasticsearch.xpack.core.security.action.apikey.CreateApiKeyRequest;
```

Comment:
I think this import is redundant

Reply:
You are very right. Spotless caught me as well 🤦